nasa/ow_autonomy
# Oceanwater 609 autonomy listen to fault msgs

Jira [609](https://babelfish.arc.nasa.gov/jira/browse/OCEANWATER-609)
summary:
* all faults messages are being listened to by Autonomy
* reports on status of these messages during any autonomy call
* report is sent only once, and then resolved message is sent when fault is removed/unselected.

Test:
* run any roslaunch file
* run any autonomy plan
* go to RQT and select any fault
* observe the system level and component level report of fault (occurs once) in RED
* unselect fault
* observe the resolve message (occurs once)  for system and component level in WHITE
